### ISSUE TITLE
Gutenberg Compatible?

### ISSUE THREAD

**Issue by salesigniter** [zedzedzed/table-of-contents-plus#152]
does it work on Gutenberg? I am not seeing the TOC and if I manually enable it with the TOC shortcode no headings / toc links are shown. My page has many h title tags so it should work.

**Comment by zedzedzed** [comment]
Yeah it should appear after you publish or preview the page, it doesn't show when you're editing it. Check the settings, by default it only shows for pages (not posts or other content types) with at least 4 headings.
